doc/api/events.md

```diff
@@ -155,9 +155,63 @@ myEmitter.emit('error', new Error('whoops!'));
 // Prints: whoops! there was an error
 ```
 
+## AutoCatch of Promise rejection
```

Comment:
I would prefer an explicit name like `captureRejections` or something that indicates that rejections are converted to errors.